### ISSUE TITLE
Non bundled chromium not launching on CentOS7

### ISSUE THREAD

**Issue by VictorChen** [puppeteer/puppeteer#1715]
**TLDR**: puppeteer works fine when using the bundled chromium, but not when I use my own installed chromium.

---

**Environment:**

* Puppeteer version: v0.13.0
* Platform / OS version: CentOS7
* Node.js version: v8.8.1

I installed chromium from this rpm:
https://centos.pkgs.org/7/epel-x86_64/chromium-61.0.3163.100-1.el7.x86_64.rpm.html

I set the `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` environment variable to avoid having to download the bundled chromium. Here's my simple script:

```js
const puppeteer = require('puppeteer');

(async () => {
  const browser = await puppeteer.launch({
    executablePath: '/bin/chromium-browser',
    dumpio: true
  });
  const page = await browser.newPage();
  await page.goto('https://example.com');
  await page.screenshot({path: 'example.png'});
  browser.close();
})();
```

output:

```
[0104/044555.684432:ERROR:devtools_http_handler.cc(786)]
DevTools listening on 127.0.0.1:36370

(node:8370) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Timed out after 30000 ms while trying to connect to Chrome! The only Chrome revision guaranteed to work is r515411
(node:8370) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

seems like puppeteer is expecting a different format?

https://github.com/GoogleChrome/puppeteer/blob/24354a4879849c8cb4e50cfcae83d78f65938a93/lib/Launcher.js#L245

**Comment by aslushnikov** [comment]
Chrome 61 is too old for puppeteer 0.13. Please try either a newer version of chromium or older version of puppeteer (v0.9.0 might work).

**Comment by VictorChen** [comment]
Thanks, FYI this worked for me (with puppeteer 0.12.0):

http://dl.google.com/linux/chrome/rpm/stable/x86_64//google-chrome-stable-63.0.3239.108-1.x86_64.rpm
